deployments/examples/ocis_wopi/docker-compose.yml

```diff
@@ -84,7 +84,10 @@ services:
       NOTIFICATIONS_SMTP_USERNAME: notifications@${OCIS_DOMAIN:-ocis.owncloud.test}
       NOTIFICATIONS_SMTP_INSECURE: "true" # the mail catcher uses self signed certificates
       # make the registry available to the app provider containers
-      MICRO_REGISTRY: "mdns"
+      MICRO_REGISTRY: "natsjs"
```

Comment:
IIRC we want to default to `nats-js-kv`

Reply:
That is true, I've deleted the `MICRO_REGISTRY` line and it works fine.
Please review again